listenbrainz/mbid_mapping/push.sh

```diff
@@ -10,5 +10,5 @@
 
```

Comment:
Is this file still needed? (that is, do you use it to push the image to docker hub?) If so, please update the comment on line 4 (not listenbrainz) and line 5 (not 'latest')

Reply:
Yep. Will update.